jinahub/encoders/image/CLIPImageEncoder/clip_image.py

```diff
@@ -41,6 +21,24 @@ def __init__(
         *args,
         **kwargs,
     ):
+        """
+        :param pretrained_model_name_or_path: Can be either:
+        - A string, the model id of a pretrained CLIP model hosted
+            inside a model repo on huggingface.co, e.g., 'openai/clip-vit-base-patch32'
+        - A path to a directory containing model weights saved, e.g.
+            `./my_model_directory/`
+        :param base_feature_extractor: Base feature extractor for images.
+            Defaults to ``pretrained_model_name_or_path`` if None
+        :param use_default_preprocessing: Whether to use the `base_feature_extractor` on
+            images (blobs) before encoding them. If you disable this, you must ensure
+            that the images you pass in have the correct format, see the ``encode``
+            method for details.
+        :param device: Pytorch device to put the model on, e.g. 'cpu', 'cuda', 'cuda:1'
+        :param default_traversal_paths: Default traversal paths for encoding, used if
+            the traversal path is not passed as a parameter with the request.
+        :param default_batch_size: Default batch size for encoding, used if the
+            batch size is not passed as a parameter with the request.

```

Comment:
For the featured ones, I suggest renaming them to `traversal_paths` and `batch_size`. The prefix `default_` is verbose because we are setting default parameters in the `__init__` anyway.

Reply:
I think this is a pattern we follow in all executors and we might not want to add some inconsistency with other executors

Reply:
I agree with @nan-wang here, and also I think it's ok if we start changing it in some executors - I've discovered that large sweeping changes across all executors are not worth it.